### PR TITLE
Add virtual to be able to override apiclient and add new ctor

### PR DESCRIPTION
**Pull request by martijn00**

### CODE REVIEW

File: src/Xablu.WebApiClient/WebApiClient.cs [hunk 1]
```diff
@@ -22,12 +22,10 @@ public class WebApiClient : IWebApiClient
         private Lazy<HttpClient> _background;
         private Lazy<HttpClient> _userInitiated;
         private Lazy<HttpClient> _speculative;
-        private Func<HttpMessageHandler> httpHandler;
+        private Func<HttpMessageHandler> httpHandler = () => new HttpClientHandler();
```
Comment (mvanbeusekom) [inline src/Xablu.WebApiClient/WebApiClient.cs]: Just nit picking but I suggests to prefix private class fields with an underscore (so rename "httpHandler" to "_httpHandler"), so it matches with the other private fields (same convention).

Comment (martijn00) [inline src/Xablu.WebApiClient/WebApiClient.cs]: Right, good suggestion. I just didn't touch that for now because it already was like that.

**Comment by MarcBruins** [comment]
Can we publish a nuget version upgrade after this merge?

File: src/Xablu.WebApiClient/WebApiClient.cs (hunk 1 above)
Comment (mvanbeusekom) [inline src/Xablu.WebApiClient/WebApiClient.cs]: Yeah I noticed, that's why I mentioned its just nit picking. But good you fixed it (all about being a good boyscout ;) )

**Pull request merged**